bottles/backend/health.py

```diff
@@ -104,38 +93,33 @@ def check_x11(self):
 
     @staticmethod
     def check_wayland():
-        if "WAYLAND_DISPLAY" in os.environ:
-            return True
-        return False
+        return "WAYLAND_DISPLAY" in os.environ
 
     def check_xwayland(self):
-        if self.x11 and self.wayland:
-            return True
-        return False
+        return self.x11 and self.wayland
 
     def check_desktop(self):
         return os.environ.get("DESKTOP_SESSION")
```

Comment:
```suggestion
        return os.environ.get("XDG_CURRENT_DESKTOP")
```

`DESKTOP_SESSION` is deprecated, and has been for a while.